File: Lista 1/q1.java
import java.util.Scanner;

public class q1 {
    public static void main(String[] args){
        double sum = 0, variacao = 0;
        final double [] medicao = new double[10];

        Scanner scanner = new Scanner(System.in);

        for(int i = 0; i < medicao.length; i++){
            
            System.out.printf("Medicao %d: ", i);
            double medido = scanner.nextDouble();

            medicao[i] = medido;
            sum += medido;
        }

        double media = sum/medicao.length;

        for(int i = 0; i < medicao.length; i++){
            variacao += Math.pow(medicao[i] - media, 2);
            variacao /= medicao.length; 
        }

        double desvio = Math.sqrt(variacao);

        System.out.println("Variacao: " + variacao);
        System.out.println("Desvio padrao: " + desvio);

        if(media * 0.1 < desvio){
            System.out.println("O multimetro esta com problemas.");
        }else{
            System.out.println("O multimetro esta funcionando.");
        }
        

    }
}

File: Lista 2/q1.py
from math import sqrt

total = 0
medicao = []
variacao = 0

for i in range(10):
    medido = float(input(f"Medicao {i}: "))

    medicao.append(medido)
    total += medido
    
media = total/len(medicao)

for j in medicao:
    variacao += (j - media) ** 2
    
variacao /= len(medicao)

desvio = sqrt(variacao)

print(f"Media: {media}")
print(f"desvio: {desvio}")

if desvio > variacao * 0.1:
    print("O multimetro esta com defeito.")
else:
    print("O multimetro esta funcionando.")
